metalearn/metafeatures/metafeatures.py

```diff
@@ -134,45 +145,47 @@ def check_time():
 
         if column_types is None:
             column_types = self._infer_column_types(X, Y)
-        if metafeature_ids is None:
-            metafeature_ids = self._get_metafeature_ids(exclude)
-            exclude = None
+        metafeature_ids = self._get_metafeature_ids(metafeature_ids, exclude, groups, exclude_groups)
```

Comment:
in some places, the api has `group` first, `exclude` second. Here it is reversed. Let's make sure the api is consistent everywhere so we don't have any accidental mixups.

Reply:
Documenting this: https://github.com/byu-dml/metalearn/issues/210